code/obj/artifacts/artifact_objects/cloner.dm

```diff
@@ -0,0 +1,105 @@
+
+/obj/artifact/cloner
+	name = "artifact cloner"
+	associated_datum = /datum/artifact/cloner
+
+/datum/artifact/cloner
+	associated_object = /obj/artifact/cloner
+	rarity_class = 4
+	min_triggers = 2
+	max_triggers = 2
+	validtypes = list("ancient","martian","wizard","eldritch","precursor")
+	validtriggers = list(/datum/artifact_trigger/carbon_touch,/datum/artifact_trigger/silicon_touch)
+	react_xray = list(15,90,90,11,"HOLLOW")
+	validtypes = list("wizard","eldritch")
+	touch_descriptors = list("You seem to have a little difficulty taking your hand off its surface.")
+	var/mob/living/carbon/human/clone = null
+	var/imprison_time = 0
+	var/evil_delay = 0
+	var/swapSouls = FALSE
+
+	New()
+		..()
+		imprison_time = rand(50,1200)
+		evil_delay = rand(0,imprison_time)
+		swapSouls = pick(TRUE, FALSE)
+
+	effect_touch(var/obj/O,var/mob/living/user)
+		if (..())
+			return
+		if (!user)
+			return
+		if (clone)
+			return
+		if (ishuman(user))
+			var/mob/living/carbon/human/H = user
+			// fluff
+			if(swapSouls)
+				boutput(user, "<span class='alert'>You feel your soul being sucked out of your body by [O]!</span>")
+			var/filter = filter(type="outline", size=0.5, color=rgb(255,0,0,128), flags=OUTLINE_SHARP)
+			H.filters += filter
+			SPAWN_DBG(0.7 SECONDS)
+				H.filters -= filter
+
+			// a bunch of stolen cloner code
+			clone = new /mob/living/carbon/human/clone(O)
+			clone.bioHolder.CopyOther(H.bioHolder, copyActiveEffects = TRUE)
+			clone.set_mutantrace(H.bioHolder?.mobAppearance?.mutant_race?.type)
+			clone.update_colorful_parts()
+			if (H.abilityHolder)
+				clone.abilityHolder = H.abilityHolder.deepCopy()
+				clone.abilityHolder.transferOwnership(clone)
+				clone.abilityHolder.remove_unlocks()
+			if(H.traitHolder && H.traitHolder.traits.len)
+				clone.traitHolder.traits = H.traitHolder.traits.Copy()
+			clone.real_name = user.real_name
+
+			if(clone.client) // gross hack for resetting tg layout bleh bluh copied from cloner code
+				clone.client.set_layout(clone.client.tg_layout)
+
+			if(swapSouls && H.mind)
+				H.mind.transfer_to(clone)
+			clone.changeStatus("paralysis", imprison_time) // so they don't ruin the surprise
+
+			if ((ticker?.mode && istype(ticker.mode, /datum/game_mode/revolution)) && ((clone.mind in ticker.mode:revolutionaries) || (clone.mind in ticker.mode:head_revolutionaries)))
+				ticker.mode:update_all_rev_icons() //So the icon actually appears
+
+			ArtifactLogs(user, clone, O, "touched","creating an evil clone of ([user])",0)
+			if(swapSouls)
+				// make original body evil
+				H.attack_alert = 0
+				H.ai_init()
+				SPAWN_DBG(0)
+					sleep(rand(1 SECOND, 10 SECONDS))
+					if(H) // completely convincing dialogue
+						H.say(pick(
+							"Well, that was weird!",
+							"Huh",
+							"Maybe it's a [pick("healer","teleporter","plant helper")] type artifact?",
+							"What do you think it does?",
+							"That activated it, didn't it?",
+							"I don't feel any different.",
+							""))
```

Comment:
Does the empty message result in `Foo Barson says ""`?

Reply:
You know, I am not sure, I didn't try it...
I will.

Reply:
I tried it, and I think it seems to not say anything, as intended by me!